File: 1013/1013 - O Maior.lua
abc = {}
for i in string.gmatch(io.read() , "%S+") do
  table.insert(abc, tonumber(i))
end
maior = (abc[1] + abc[2] + math.abs(abc[1] - abc[2]))/2
maior2 = (maior + abc[3] + math.abs(maior - abc[3]))/2
print(string.format("%.0f eh o maior", maior2))

File: 1013/1013 - O Maior.cs
using System;
class Program
{
    static void Main(string[] args) {
        int a,b,c,maior,maior2;
        var abc = Console.ReadLine().Split();
        a = Convert.ToInt16(abc[0]);
        b = Convert.ToInt16(abc[1]);
        c = Convert.ToInt16(abc[2]);
        maior = (a + b + Math.Abs(a - b))/2;
        maior2 = (maior + c + Math.Abs(maior - c))/2;
        Console.WriteLine("{0} eh o maior", maior2.ToString("F0"));
    }
}
